metsrw/fsentry.py

```diff
@@ -502,6 +502,9 @@ def get_premis_rights(self):
 
     def get_premis_rights_statement(self, rights_statement_uuid):
         for rights in self.get_premis_rights():
-            if rights.rights_statement_identifier_value == rights_statement_uuid:
+            if (
```

Comment:
```
for statement in rights.rights_statement:
    if statement.rights_statement_identifier_value == rights_statement_uuid:
        return rights
```
(minor though, we don't write `rights` elements with multiple `rightsStatement` elements).